src/Http/Headers/HttpClientAcceptHeader.ts

```diff
@@ -0,0 +1,8 @@
+import HttpClientHeader from "./HttpClientHeader";
+import IHttpClientHeader from "./IHttpClientHeader";
+
+export class HttpClientAcceptHeader extends HttpClientHeader implements IHttpClientHeader {
```

Comment:
This might be redundant on a whole, perhaps just use a constant for the Header name and use `HttpClientHeader`.